SwiftLint/copy-and-run-swiftlint-config.sh

```diff
@@ -96,7 +96,11 @@ rm "$temporarySwiftLintConfigFilename"
 
 cd "$projectDir"
 
-SWIFTLINT_EXECUTABLE="$projectDir/Pods/SwiftLint/swiftlint"
+if [ -e "$projectDir/Pods/SwiftLint" ]; then
+	echo "warning: SwiftLint should not be added as Pod, as it is already located in the submodules folder"
```

Comment:
There might be many submodules. -> `... located in SMF-iOS-CommonProjectSetupFiles.`